### PR TITLE
Fix "Focus on Following/Unfollow button from the right Sidebar"

### LINKED ISSUE #1188: The Following/Unfollow button from the right menu of profile page is glitchy
Copied from Utopian.

<div class="Body Body--full"><div><p>In this post I want to report a bug I found on the <a href="https://busy.org" target="_blank" rel="nofollow noopener">busy</a> website.</p>
<p>While I was on my profile page, I noticed that the Following/Unfollow button from the right menu is glitchy. Precisely, when the mouse cursor is positioned on the left edge of the button, the behavior mentioned above is taking place.</p>
<p>I observed this issue, on my laptop which has a resolution: 1366 x 768.</p>
<p>For a better overview, I made the gif bellow:</p>
<p></p>
<p>I changed the site’s language to Romanian, to see if the issue is present. As you can see, from the gif bellow, when we switch to Romanian the problem disappears:</p>
<p></p>
<p>I also changed the site’s language to German. The glitchy behavior is absent here:</p>
<p></p>
<p>To test I used Google Chrome 62, on Windows 10.</p>
<p><br></p><hr><em>Posted on <a href="https://utopian.io/utopian-io/@sarmizegetusa/ui-bug-the-following-unfollow-button-from-the-right-menu-of-profile-page-is-glitchy" target="_blank" rel="nofollow noopener">Utopian.io -  Rewarding Open Source Contributors</a></em><hr><p></p></div></div>

### PR DESCRIPTION
**Pull request by sweever**
Fixes #1188

fixing "hover" action on follow button on right sidebar.
Changes:

- Changes on : src/client/components/Sidebar/InterestingPeople.less
### Add Scale insted of  Padding ;
```
transform: scale(.9);
```

- Changes on ; src/client/components/Sidebar/User.less
### updating user_follow Class  ;
- Set 40% **width** to button container 
- Set 100% **width** to button  
```
.User__follow {
  width: 40%;
  vertical-align: middle;
  margin-left: auto;
  
  .Follow {
	width: 100%!important;
  }
}
```
2umkfjc6ni
xbeownd260
zdkatabesm

### CODE REVIEW

**Comment by bonustrack** [comment]
I don't think we should use strict width on these buttons, the space for username is already too small these changes make it smaller.

File: src/client/components/Sidebar/User.less [hunk 1]
```diff
@@ -28,9 +28,13 @@
 }
 
 .User__follow {
-  display: flex;
+  width: 40%;
```
Comment (jm90m) [inline src/client/components/Sidebar/User.less]: We shouldn't force a 40% width on a button, it should be dynamic of the container and contents.

File: src/client/components/Sidebar/InterestingPeople.less [hunk 2]
```diff
@@ -7,7 +7,7 @@
 }
 
 .InterestingPeople__container {
-  padding: 8px 16px 16px 16px;
+  transform: scale(.9);
```
Comment (jm90m) [inline src/client/components/Sidebar/InterestingPeople.less]: why this and remove the current padding? The padding is similar to all our other sidebar components.

Comment (sweever) [reply src/client/components/Sidebar/InterestingPeople.less]: padding take more space instead of scale.

**Comment by bonustrack** [comment]
I'm closing this PR, i dont think it's the right way to fix with issue.

**Pull request closed without merging**